frontend/src/pages/NutriScore.jsx

```diff
@@ -6,57 +6,49 @@ function NutriScore() {
     <div className="main-div">
       <div className="text-container">
         <h2 className="text-title">
-          Un Système d'Étiquetage Nutritionnel qui Simplifie le Choix
-          Alimentaire
+          A Nutrition Labeling System That Simplifies Choice Eating
         </h2>
         <p className="text-description">
-          Les valeurs nutritionnelles jouent un rôle essentiel dans notre
-          compréhension de la qualité nutritionnelle des produits alimentaires.
-          Le Nutri-Score, par exemple, est un système d'étiquetage nutritionnel
-          qui utilise ces valeurs pour attribuer une note globale à un produit
-          alimentaire, facilitant ainsi la prise de décision des consommateurs.
-          En effet, le Nutri-Score met en évidence la valeur nutritionnelle d'un
-          produit de manière simple et visuelle, allant de la lettre A (vert)
-          pour les produits les plus sains à la lettre E (rouge) pour les moins
-          sains. Ce système aide les consommateurs à choisir des aliments plus
-          équilibrés et à opter pour des choix plus sains, en un coup d'œil sur
-          l'étiquette.
+          Nutritional values play an essential role in our understanding the
+          nutritional quality of food products. Nutri-Score, for example, is a
+          nutritional labeling system which uses these values to assign an
+          overall rating to a product food, thus facilitating consumer
+          decision-making. Indeed, the Nutri-Score highlights the nutritional
+          value of a produced in a simple and visual way, going from the letter
+          A (green) for the healthiest products with the letter E (red) for the
+          least healthy. This system helps consumers choose foods more balanced
+          and opt for healthier choices, at a glance the label.
```

Comment:
Cadeau => Nutritional values play an essential role in our understanding of the nutritional quality of food products. Nutri-Score, for example, is a nutritional labeling system which uses these values to assign an overall rating to a product food, thus facilitating consumer decision-making. Indeed, the Nutri-Score highlights the nutritional value of a produced in a simple and visual way, going from the letter A (green) for the healthiest products with the letter E (red) for the least healthy. This system helps consumers choose foods more balanced and opt for healthier choices, at a glance at the label.